uk/lessons/basics/basics.md

```diff
@@ -113,7 +113,8 @@ iex> :true === true
 true
 ```
 
-Назви модулів в Elixir також є атомами. `MyApp.MyModule` це валідний атом, навіть якщо такий модуль ще не був заявлений:
+Назви модулів в Elixir також є атомами.
+`MyApp.MyModule` це валідний атом, навіть якщо такий модуль ще не був заявлений:
```

Comment:
Why add extra blank lines? They are not in the original..

Reply:
Oops, I guess I got a little carried away with all the newlining